dataland-frontend/src/components/forms/parts/kpiSelection/FinancialShareFormField.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <FormKit type="group" :name="name">
+    <div class="mb-3 form-field">
+      <UploadFormHeader :label="label" :description="description ?? ''" :is-required="required" />
+      <div class="next-to-each-other">
+        <FormKit
+          type="text"
+          name="relativeShareInPercent"
+          validation-label="Relative Value"
+          validation="number|between:0,100"
+          placeholder="Relative Value in %"
+          outer-class="short"
+          data-test="relativeShareInPercent"
+        />
+        <FormKit type="group" name="absoluteShare">
+          <FormKit
+            type="text"
+            name="amount"
+            validation-label="Absolute Value"
+            validation="number"
+            placeholder="Absolute Value"
+            outer-class="short"
+            data-test="absoluteShareAmount"
+          />
+          <SingleSelectFormElement
+            name="currency"
+            validation="length:2,3"
```

Comment:
What is this needed for? Don't all currencies have a length between 2 and 3?

Reply:
removed